marimo/_server/sessions.py

```diff
@@ -661,6 +661,7 @@ def __init__(
         cli_args: SerializedCLIArgs,
         auth_token: Optional[AuthToken],
         redirect_console_to_browser: bool = False,
+        global_session: bool = False,
```

Comment:
maybe we can rename to `global_session_in_run_mode` or something similar. it is verbose, but this could be confused that edit mode also as global session configuration

Reply:
Yeah certainly, any name that makes sense is good for me :)